Resources/public/js/views/services/plugins/ez-copycontentplugin.js

```diff
@@ -0,0 +1,143 @@
+         *
+         * @method _copySelectLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _copySelectLocation: function (e) {
+            var service = this.get('host');
+
+            service.fire('contentDiscover', {
+                config: {
+                    title: "Select the location you want to copy your content into",
+                    contentDiscoveredHandler: Y.bind(this._copyContent, this),
+                },
+            });
+        },
+
+        /**
+         * Copy the content to the selected location
+         *
+         * @method _copyContent
+         * @protected
+         * @param {EventFacade} e
+         */
+        _copyContent: function (e) {
+            var service = this.get('host'),
+                app = service.get('app'),
+                parentLocationId = e.selection.location.get('id'),
+                locationId = service.get('location').get('id'),
+                that = this,
+                contentName =  service.get('content').get('name'),
+                parentContentName = e.selection.content.get('name'),
+                notificationIdentifier =  'copy-notification-' + parentLocationId + '-' + locationId;
+
+            this._notify(
+                "'" + contentName + "' is being copied under '" + parentContentName + "'",
+                notificationIdentifier,
+                'started',
+                5
+            );
+            app.set('loading', true);
+            service.get('content').copy({api: service.get('capi')}, parentLocationId, function (error, response) {
+                if (error) {
+                    that._notify('An error occured while copying your content', notificationIdentifier, 'error', 0);
+                    app.set('loading', false);
+                    return;
+                }
+                that._notify(
+                    "'" + contentName + "' has been successfully copied under '" + parentContentName + "'",
+                    notificationIdentifier,
+                    'done',
+                    5
+                );
+                that._redirect(response.getHeader('location'));
+            });
+        },
+
+        /**
+         * Redirect the user to the copied content
+         *
+         * @method _redirect
+         * @protected
+         */
+        _redirect: function (id) {
+            var service = this.get('host'),
+                app = service.get('app'),
+                that = this;
+
+            this.get('copiedContent').set('id', id);
+            this.get('copiedContent').load({api: service.get('capi')}, function (error) {
+                if (error) {
+                    app.set('loading', false);
+                    that._notify('An error occured while loading your content', 'loading-content-error', 'error', 0);
+                    return;
+                }
+                app.navigateTo('viewLocation', {id: that.get('copiedContent').get('resources').MainLocation});
```

Comment:
to avoid some work to @yannickroger in https://github.com/ezsystems/PlatformUIBundle/pull/233/ please add the `languageCode` to the parameters of the route.